include/lsst/afw/image/CoaddInputs.h

```diff
@@ -62,6 +62,15 @@ class CoaddInputs : public table::io::PersistableFacade<CoaddInputs>, public tab
     /// Construct from shallow copies of the given catalogs.
     CoaddInputs(table::ExposureCatalog const & visits_, table::ExposureCatalog const & ccds_);
 
+    // This code is temporary diagnostic code; an attempt to work around
+    // a pointer error seen in python when trying to read "visits" (or, presumably, "ccds").
+    // These added methods allow one to wrap access using def_property instead of def_readwrite.
+    // Once the problem is resolved please remove this code.
+    table::ExposureCatalog & _getVisits() { return visits; };
+    table::ExposureCatalog & _getCcds() { return ccds; };
+    void _setVisits(table::ExposureCatalog & visits) { this->visits = visits; };
+    void _setCcds(table::ExposureCatalog & ccds) { this->ccds = ccds; };
```

Comment:
Almost. I figured it might help you diagnose DM-8497, but I want to remove it once DM-8497 is fixed. The current pybind11 wrapper (coaddInputs.cc) has an if/else that will use the usual `def_readwrite` to wrap `exposures` and `ccds` directly (ignoring these new methods), but that can be toggled to use `def_property` and use these new methods. Neither works, so it wasn't much help, but I left it in case you find it of any use.

Note that this extra code was added as a special separate commit. My plan is to merge this ticket without that extra commit, but leave the commit on the ticket branch.